ROADMAP.md

```diff
@@ -0,0 +1,94 @@
+# io.js Roadmap
+
+***This is a living document, it describes our policy and priorities as they exist today but can evolve over time.***
+
+## Stability Policy
+
+The single more important consideration in every code change is the impact it will have, positive or negative, on the ecosystem (modules and applications).
+
+io.js does not remove JavaScript API.
```

Comment:
There are two points to consider here. I believe the intent is that "io.js does not remove JavaScript API from core". Beyond that, there's the interesting challenge that might occur dealing with volatile changes in V8 such as the broken Promises and Classes that shipped at one point.